### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by str** [holger1411/understrap-child#39]
In an ubuntu server 16.10 I ran `npm install` and it returns the following error message:

```
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.17
npm ERR! Linux 4.8.6-x86_64-linode78
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! node-sass@4.5.0 install: `node scripts/install.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the node-sass@4.5.0 install script 'node scripts/install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-sass
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.
```

[npm-debug.txt](https://github.com/holger1411/understrap-child/files/776668/npm-debug.txt)

**Comment by holger1411** [comment]
Seems to be a npm bug. Found a nearly identical issue over there at the npmissue tracker:
https://github.com/npm/npm/issues/13110

There are some solutions described below the issue, but the bug is still open.
You could also update node (including npm)

**Comment by str** [comment]
I tested it today with latest version. The error does not happen anymore.

**Comment by amdrade** [comment]
@holger1411 
I'm try to install and returned the following message:

`No compatible version found: understrap@^0.6.1`

Linux 4.10.0-21-generic
node v7.9.0
npm  v4.2.0

**Comment by holger1411** [comment]
@amdrade: Sorry, I released the new 0.6.1 version yesterday but publish it via npm this morning.
Now it should be avialable.

**Comment by amdrade** [comment]
Ok, @holger1411 Thank you! 
Congratulations on the project
